src/styles/page-template/_template.njk

```diff
@@ -12,7 +12,8 @@
 
 {% if pageConfig.cdn or release_version %}
     {# Production #}
-    {% set assetsURL = "https://cdn.ons.gov.uk/sdc/design-system/" + (pageConfig.cdn or release_version) %}
+    {% set cdn_url = pageConfig.cdn.url or "https://cdn.ons.gov.uk/sdc/design-system/" %}
+    {% set assetsURL = cdn_url + (pageConfig.cdn.version or release_version) %}
```

Comment:
@rmccar @boxadesign Are you aware of any services that are using `pageConfig.cdn` at the moment for the `version`?

Reply:
Yes so this would be a breaking change which we can handle ok.

Reply:
When would we want to use a different version of the CDN assets to the template `release_version`?

Reply:
We wouldn't but others might for whatever reason. I'm just supporting what was already supported. Previously the var was just called `cdn` which is misleading.

Reply:
@boxadesign just to confirm, other services make use of the `cdn version`?. For any given version of templates you load, the `release_version` would match. Would you happened to know of the use case for this?

Reply:
So the `release_version` gets set if you use the design system components via the zip that gets generated on release. You can of course use the DS is a few other ways i.e. NPM or manually and you can set the cdn version via the `pageConfig`